pithos/pandora/pandora.py

```diff
@@ -229,8 +231,29 @@ def connect(self, client, user, password):
         self.userId = user['userId']
         self.userAuthToken = user['userAuthToken']
 
+        self.connected = True
         self.get_stations(self)
 
+    @property
+    def explicit_content_filter_state(self):
+        """The User must already be authenticated before this is called.
+           returns the state of Explicit Content Filter and if the Explicit Content Filter is PIN protected
+        """
+        get_filter_state = self.json_call('user.getSettings', https=True)
+        filter_state = get_filter_state['isExplicitContentFilterEnabled']
+        pin_protected = get_filter_state['isExplicitContentFilterPINProtected']
+        logging.info('Explicit Content Filter state: %s', filter_state)
+        logging.info('PIN protected: %s', pin_protected)
+        return filter_state, pin_protected
+
+    def set_explicit_content_filter_to(self, state):
```

Comment:
Minor nit-pick, but to be consistent remove `_to()` from the setter to be consistent with others.